tests/appsec/app.py

```diff
@@ -220,8 +220,8 @@ def iast_ast_patching_re():
     return resp
 
 
-@app.route("/iast-ast-patching-non-re", methods=["GET"])
-def iast_ast_patching_non_re():
+@app.route("/iast-ast-patching-non-re-sub", methods=["GET"])
+def iast_ast_patching_non_re_sub():
```

Comment:
`iast_ast_patching_re_sub` and `iast_ast_patching_non_re_sub` are the same function, are they?

why not?

@app.route("/iast-ast-patching-re-sub", methods=["GET"])
@app.route("/iast-ast-patching-non-re-sub", methods=["GET"])
def iast_ast_patching_re_sub():
   ...

Reply:
They're not the same, non-re-sub does this extra line at the top of the function:
```
    import iast.fixtures.non_re_module as re
```

Reply:
We could do that under an if depending on some request.args though

Reply:
Tested with the if, it doesn't work as expected